src/gtfs/write.rs

```diff
@@ -39,8 +39,10 @@ pub fn write_transfers(path: &path::Path, transfers: &Collection<NtfsTransfer>)
     let mut wtr =
         csv::Writer::from_path(&path).with_context(|_| format!("Error reading {:?}", path))?;
     for t in transfers.values() {
-        wtr.serialize(Transfer::from(t))
-            .with_context(|_| format!("Error reading {:?}", path))?;
+        if t.from_stop_id != t.to_stop_id {
```

Comment:
I don’t really remember this stuff, but don’t you want to remove those transfer at the model loading?

Reply:
We want to keep them in ntfs, navitia uses them